fw/http.c

```diff
@@ -1076,6 +1117,123 @@ tfw_h1_send_resp(TfwHttpReq *req, int status)
 	tfw_http_resp_build_error(req);
 }
 
```

Comment:
There are already `tfw_h2_prep_redirect()` and `tfw_h1_prep_redirect()`, so please comment the new redirection functions (allocating a new responses) and these two old ones (rewriting server responses) about the differences.